common/lib/capa/capa/templates/jsinput.html

```diff
@@ -52,6 +52,6 @@
   % endif
 
   % if msg:
-      <span class="message">${msg|n}</span>
+      <span class="message" aria-describedby="iframe_${id}" aria-live="polite" tabindex="-1">${HTML(msg)}</span>
```

Comment:
@cptvitamin said that jsinput changes should be done as part of @clrux's work to create an accessible example.

@clrux let's talk when you get back.

@alisan617 go ahead and remove the aria-describedby here. Mark said that it doesn't make sense to point to an iframe.